
## by karton4ik

from PyQt6 import QtCore, QtGui, QtWidgets


class Ui_Dialog_Contactus(object):
    def setupUi(self, Dialog_Contactus):
        Dialog_Contactus.setObjectName("Dialog_Contactus")
        Dialog_Contactus.resize(1000, 666)
        Dialog_Contactus.setMinimumSize(QtCore.QSize(1000, 666))
        Dialog_Contactus.setMaximumSize(QtCore.QSize(1000, 666))
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("Images/logo.png"), QtGui.QIcon.Mode.Normal, QtGui.QIcon.State.Off)
        Dialog_Contactus.setWindowIcon(icon)
        self.label_photo = QtWidgets.QLabel(Dialog_Contactus)
        self.label_photo.setGeometry(QtCore.QRect(0, 0, 1000, 666))
        self.label_photo.setText("")
        self.label_photo.setPixmap(QtGui.QPixmap("Images/contact_us.png"))
        self.label_photo.setScaledContents(True)
        self.label_photo.setObjectName("label_photo")
        self.label_by = QtWidgets.QLabel(Dialog_Contactus)
        self.label_by.setGeometry(QtCore.QRect(80, 60, 161, 31))
        palette = QtGui.QPalette()
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 255))
        brush.setStyle(QtCore.Qt.BrushStyle.SolidPattern)
        palette.setBrush(QtGui.QPalette.ColorGroup.Active, QtGui.QPalette.ColorRole.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 255))
        brush.setStyle(QtCore.Qt.BrushStyle.SolidPattern)
        palette.setBrush(QtGui.QPalette.ColorGroup.Inactive, QtGui.QPalette.ColorRole.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(120, 120, 120))
        brush.setStyle(QtCore.Qt.BrushStyle.SolidPattern)
        palette.setBrush(QtGui.QPalette.ColorGroup.Disabled, QtGui.QPalette.ColorRole.WindowText, brush)
        self.label_by.setPalette(palette)
        font = QtGui.QFont()
        font.setFamily("Montserrat SemiBold")
        font.setPointSize(18)
        font.setBold(True)
        font.setWeight(75)
        self.label_by.setFont(font)
        self.label_by.setObjectName("label_by")
        self.label_email = QtWidgets.QLabel(Dialog_Contactus)
        self.label_email.setGeometry(QtCore.QRect(80, 590, 381, 31))
        palette = QtGui.QPalette()
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 255))
        brush.setStyle(QtCore.Qt.BrushStyle.SolidPattern)
        palette.setBrush(QtGui.QPalette.ColorGroup.Active, QtGui.QPalette.ColorRole.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 255))
        brush.setStyle(QtCore.Qt.BrushStyle.SolidPattern)
        palette.setBrush(QtGui.QPalette.ColorGroup.Inactive, QtGui.QPalette.ColorRole.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(120, 120, 120))
        brush.setStyle(QtCore.Qt.BrushStyle.SolidPattern)
        palette.setBrush(QtGui.QPalette.ColorGroup.Disabled, QtGui.QPalette.ColorRole.WindowText, brush)
        self.label_email.setPalette(palette)
        font = QtGui.QFont()
        font.setFamily("Montserrat SemiBold")
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.label_email.setFont(font)
        self.label_email.setTextInteractionFlags(QtCore.Qt.TextInteractionFlag.LinksAccessibleByMouse|QtCore.Qt.TextInteractionFlag.TextSelectableByMouse)
        self.label_email.setObjectName("label_email")
        self.label_github = QtWidgets.QLabel(Dialog_Contactus)
        self.label_github.setGeometry(QtCore.QRect(80, 570, 341, 31))
        palette = QtGui.QPalette()
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 255))
        brush.setStyle(QtCore.Qt.BrushStyle.SolidPattern)
        palette.setBrush(QtGui.QPalette.ColorGroup.Active, QtGui.QPalette.ColorRole.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 255))
        brush.setStyle(QtCore.Qt.BrushStyle.SolidPattern)
        palette.setBrush(QtGui.QPalette.ColorGroup.Inactive, QtGui.QPalette.ColorRole.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(120, 120, 120))
        brush.setStyle(QtCore.Qt.BrushStyle.SolidPattern)
        palette.setBrush(QtGui.QPalette.ColorGroup.Disabled, QtGui.QPalette.ColorRole.WindowText, brush)
        self.label_github.setPalette(palette)
        font = QtGui.QFont()
        font.setFamily("Montserrat SemiBold")
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.label_github.setFont(font)
        self.label_github.setMouseTracking(False)
        self.label_github.setOpenExternalLinks(False)
        self.label_github.setTextInteractionFlags(QtCore.Qt.TextInteractionFlag.LinksAccessibleByMouse|QtCore.Qt.TextInteractionFlag.TextSelectableByMouse)
        self.label_github.setObjectName("label_github")
        self.label_github.setOpenExternalLinks(True)
        self.retranslateUi(Dialog_Contactus)
        
        
        
        QtCore.QMetaObject.connectSlotsByName(Dialog_Contactus)
        
    def retranslateUi(self, Dialog_Contactus):
        _translate = QtCore.QCoreApplication.translate
        Dialog_Contactus.setWindowTitle(_translate("Dialog_Contactus", "Contact us"))
        self.label_by.setText(_translate("Dialog_Contactus", "By karton4ik"))
        self.label_email.setText(_translate("Dialog_Contactus", "e-mail: tonchik.kartonchik@gmail.com"))
        self.label_github.setText(_translate("Dialog_Contactus", "github: <a href='https://github.com/cardboardchik/'>github.com/cardboardchik</a>"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    Dialog_Contactus = QtWidgets.QDialog()
    ui = Ui_Dialog_Contactus()
    ui.setupUi(Dialog_Contactus)
    Dialog_Contactus.show()
    sys.exit(app.exec())
